include/olt-manager.php

```diff
@@ -85,155 +60,43 @@ public static function instance() {
-	 * It replaces the 'override_load_textdomain' filter previously used.
+	 * Prevents WP loading textdomains before we set the locale ourselves.
 	 *
 	 * @since 2.0.4
+	 * @since 3.6 Renamed from `load_textdomain_mofile()`.
 	 *
-	 * @param string $mofile The translation file name.
-	 * @param string $domain The text domain name.
 	 * @return string
 	 */
-	public function load_textdomain_mofile( $mofile, $domain ) {
-		// On multisite, 2 files are sharing the same domain so we need to distinguish them.
-		if ( 'default' === $domain && false !== strpos( $mofile, '/ms-' ) ) {
-			$this->list_textdomains['ms-default'] = array( 'mo' => $mofile, 'domain' => $domain );
-		} else {
-			$this->list_textdomains[ $domain ] = array( 'mo' => $mofile, 'domain' => $domain );
-		}
-		return ''; // Hack to prevent WP loading text domains as we will load them all later.
-	}
-
-	/**
-	 * Saves post types and taxonomies labels for a later usage
-	 *
-	 * @since 0.9
-	 *
-	 * @param string $translation not used
-	 * @param string $text        string to translate
-	 * @param string $domain      text domain
-	 * @return string unmodified $translation
-	 */
-	public function gettext( $translation, $text, $domain ) {
-		if ( is_string( $text ) ) { // Avoid a warning with some buggy plugins which pass an array
-			$this->labels[ $text ] = array( 'domain' => $domain );
-		}
-		return $translation;
-	}
-
-	/**
-	 * Saves post types and taxonomies labels for a later usage
-	 *
-	 * @since 0.9
-	 *
-	 * @param string $translation not used
-	 * @param string $text        string to translate
-	 * @param string $context     some comment to describe the context of string to translate
-	 * @param string $domain      text domain
-	 * @return string unmodified $translation
-	 */
-	public function gettext_with_context( $translation, $text, $context, $domain ) {
-		$this->labels[ $text ] = array( 'domain' => $domain, 'context' => $context );
-		return $translation;
-	}
-
-	/**
-	 * Translates post types and taxonomies labels once the language is known.
-	 *
-	 * @since 0.9
-	 *
-	 * @param WP_Post_Type|WP_Taxonomy $type Either a post type or a taxonomy.
-	 * @return void
-	 */
-	public function translate_labels( $type ) {
-		// Use static array to avoid translating several times the same ( default ) labels
-		static $translated = array();
-
-		foreach ( (array) $type->labels as $key => $label ) {
-			if ( is_string( $label ) && isset( $this->labels[ $label ] ) ) {
-				if ( empty( $translated[ $label ] ) ) {
-					// PHPCS:disable WordPress.WP.I18n
-					$type->labels->$key = $translated[ $label ] = isset( $this->labels[ $label ]['context'] ) ?
-						_x( $label, $this->labels[ $label ]['context'], $this->labels[ $label ]['domain'] ) :
-						__( $label, $this->labels[ $label ]['domain'] );
-					// PHPCS:enable
-				}
-				else {
-					$type->labels->$key = $translated[ $label ];
-				}
-			}
-		}
+	public function bypass_load_textdomain_mofile() {
+		return '';
```

Comment:
I guess we can remove this method and use `__return_empty_string()` instead.

Reply:
Done in beedce96.